<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="17" passed="11" failed="6" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-07-17T15:32:09 AMT" name="Default Suite" finished-at="2022-07-17T15:40:47 AMT" duration-ms="517550">
    <groups>
    </groups>
    <test started-at="2022-07-17T15:32:09 AMT" name="craftBetAutomation" finished-at="2022-07-17T15:40:47 AMT" duration-ms="517550">
      <class name="testCases.PasswordRecoveryTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:09 AMT" name="setup" finished-at="2022-07-17T15:32:13 AMT" duration-ms="4074" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="goToLoginPopUp()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:13 AMT" name="goToLoginPopUp" finished-at="2022-07-17T15:32:14 AMT" duration-ms="362" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- goToLoginPopUp -->
        <test-method signature="RecoveryPopUpLabelsPresenceVerification_Test()[pri:1, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:14 AMT" name="RecoveryPopUpLabelsPresenceVerification_Test" description="Validate on Password Recovery PopUp Title, Email/mobile or username input Label, Info Message, Help message, Text on Send Me Recovery Button" finished-at="2022-07-17T15:32:15 AMT" duration-ms="717" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RecoveryPopUpLabelsPresenceVerification_Test -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:15 AMT" name="tearDown" finished-at="2022-07-17T15:32:15 AMT" duration-ms="130" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:15 AMT" name="setup" finished-at="2022-07-17T15:32:18 AMT" duration-ms="3130" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="goToLoginPopUp()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:18 AMT" name="goToLoginPopUp" finished-at="2022-07-17T15:32:18 AMT" duration-ms="423" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- goToLoginPopUp -->
        <test-method signature="RecoveryPopUpLogoPresenceVerification_Test()[pri:2, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:18 AMT" name="RecoveryPopUpLogoPresenceVerification_Test" description="Validate on Password Recovery Pop Up Logo Presence" finished-at="2022-07-17T15:32:18 AMT" duration-ms="28" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RecoveryPopUpLogoPresenceVerification_Test -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:18 AMT" name="tearDown" finished-at="2022-07-17T15:32:18 AMT" duration-ms="142" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:18 AMT" name="setup" finished-at="2022-07-17T15:32:22 AMT" duration-ms="3176" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="goToLoginPopUp()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:22 AMT" name="goToLoginPopUp" finished-at="2022-07-17T15:32:22 AMT" duration-ms="581" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- goToLoginPopUp -->
        <test-method signature="RecoveryPopUpSupportLink_Test()[pri:3, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:22 AMT" name="RecoveryPopUpSupportLink_Test" description="Validate on Password Recovery Pop Up Support link" finished-at="2022-07-17T15:32:23 AMT" duration-ms="328" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[did not expect [https://craftbet.com/forgot-password] but found [https://craftbet.com/forgot-password]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: did not expect [https://craftbet.com/forgot-password] but found [https://craftbet.com/forgot-password]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failEquals(Assert.java:1417)
at org.testng.Assert.assertNotEqualsImpl(Assert.java:156)
at org.testng.Assert.assertNotEquals(Assert.java:2033)
at org.testng.Assert.assertNotEquals(Assert.java:2055)
at testCases.PasswordRecoveryTest.RecoveryPopUpSupportLink_Test(PasswordRecoveryTest.java:83)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RecoveryPopUpSupportLink_Test -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:23 AMT" name="tearDown" finished-at="2022-07-17T15:32:23 AMT" duration-ms="118" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:23 AMT" name="setup" finished-at="2022-07-17T15:32:27 AMT" duration-ms="3850" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="goToLoginPopUp()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:27 AMT" name="goToLoginPopUp" finished-at="2022-07-17T15:32:27 AMT" duration-ms="469" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- goToLoginPopUp -->
        <test-method signature="RecoveryPopUpPositiveTest()[pri:4, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:27 AMT" name="RecoveryPopUpPositiveTest" description="Validate on Password recovery pop up functionality of Recovery with valid credentials" finished-at="2022-07-17T15:32:28 AMT" duration-ms="1026" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RecoveryPopUpPositiveTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:28 AMT" name="tearDown" finished-at="2022-07-17T15:32:29 AMT" duration-ms="123" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:29 AMT" name="setup" finished-at="2022-07-17T15:32:32 AMT" duration-ms="3277" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="goToLoginPopUp()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:32 AMT" name="goToLoginPopUp" finished-at="2022-07-17T15:32:33 AMT" duration-ms="470" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- goToLoginPopUp -->
        <test-method signature="RecoveryPopUpNegativeTest(java.lang.String)[pri:5, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:33 AMT" name="RecoveryPopUpNegativeTest" description="Validate on Password recovery pop up functionality of Recovery with invalid credentials" data-provider="invalidRecoveryData" finished-at="2022-07-17T15:32:54 AMT" duration-ms="21021" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[2.0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at testCases.PasswordRecoveryTest.RecoveryPopUpNegativeTest(PasswordRecoveryTest.java:131)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RecoveryPopUpNegativeTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:54 AMT" name="tearDown" finished-at="2022-07-17T15:32:54 AMT" duration-ms="123" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:54 AMT" name="setup" finished-at="2022-07-17T15:32:57 AMT" duration-ms="2823" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="goToLoginPopUp()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:57 AMT" name="goToLoginPopUp" finished-at="2022-07-17T15:32:58 AMT" duration-ms="592" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- goToLoginPopUp -->
        <test-method signature="RecoveryPopUpNegativeTest(java.lang.String)[pri:5, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:32:58 AMT" name="RecoveryPopUpNegativeTest" description="Validate on Password recovery pop up functionality of Recovery with invalid credentials" data-provider="invalidRecoveryData" finished-at="2022-07-17T15:33:19 AMT" duration-ms="21523" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[23456.0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at testCases.PasswordRecoveryTest.RecoveryPopUpNegativeTest(PasswordRecoveryTest.java:131)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RecoveryPopUpNegativeTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:33:20 AMT" name="tearDown" finished-at="2022-07-17T15:33:20 AMT" duration-ms="127" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:33:20 AMT" name="setup" finished-at="2022-07-17T15:33:23 AMT" duration-ms="3143" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="goToLoginPopUp()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:33:23 AMT" name="goToLoginPopUp" finished-at="2022-07-17T15:33:24 AMT" duration-ms="483" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- goToLoginPopUp -->
        <test-method signature="RecoveryPopUpNegativeTest(java.lang.String)[pri:5, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:33:24 AMT" name="RecoveryPopUpNegativeTest" description="Validate on Password recovery pop up functionality of Recovery with invalid credentials" data-provider="invalidRecoveryData" finished-at="2022-07-17T15:33:45 AMT" duration-ms="21561" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[2.3456789E9]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at testCases.PasswordRecoveryTest.RecoveryPopUpNegativeTest(PasswordRecoveryTest.java:131)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RecoveryPopUpNegativeTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:33:46 AMT" name="tearDown" finished-at="2022-07-17T15:33:46 AMT" duration-ms="122" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:33:46 AMT" name="setup" finished-at="2022-07-17T15:33:49 AMT" duration-ms="3522" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="goToLoginPopUp()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:33:49 AMT" name="goToLoginPopUp" finished-at="2022-07-17T15:33:50 AMT" duration-ms="394" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- goToLoginPopUp -->
        <test-method signature="RecoveryPopUpNegativeTest(java.lang.String)[pri:5, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:33:50 AMT" name="RecoveryPopUpNegativeTest" description="Validate on Password recovery pop up functionality of Recovery with invalid credentials" data-provider="invalidRecoveryData" finished-at="2022-07-17T15:34:42 AMT" duration-ms="51775" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[123abc123]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RecoveryPopUpNegativeTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:34:42 AMT" name="tearDown" finished-at="2022-07-17T15:34:42 AMT" duration-ms="115" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:34:42 AMT" name="setup" finished-at="2022-07-17T15:34:46 AMT" duration-ms="3813" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="goToLoginPopUp()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:34:46 AMT" name="goToLoginPopUp" finished-at="2022-07-17T15:34:46 AMT" duration-ms="480" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- goToLoginPopUp -->
        <test-method signature="RecoveryPopUpNegativeTest(java.lang.String)[pri:5, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:34:46 AMT" name="RecoveryPopUpNegativeTest" description="Validate on Password recovery pop up functionality of Recovery with invalid credentials" data-provider="invalidRecoveryData" finished-at="2022-07-17T15:35:27 AMT" duration-ms="40796" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[0.12/34?56]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RecoveryPopUpNegativeTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:35:27 AMT" name="tearDown" finished-at="2022-07-17T15:35:27 AMT" duration-ms="124" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:35:27 AMT" name="setup" finished-at="2022-07-17T15:35:30 AMT" duration-ms="2795" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="goToLoginPopUp()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:35:30 AMT" name="goToLoginPopUp" finished-at="2022-07-17T15:35:30 AMT" duration-ms="536" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- goToLoginPopUp -->
        <test-method signature="RecoveryPopUpNegativeTest(java.lang.String)[pri:5, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:35:30 AMT" name="RecoveryPopUpNegativeTest" description="Validate on Password recovery pop up functionality of Recovery with invalid credentials" data-provider="invalidRecoveryData" finished-at="2022-07-17T15:35:52 AMT" duration-ms="22033" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[0.123456]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at testCases.PasswordRecoveryTest.RecoveryPopUpNegativeTest(PasswordRecoveryTest.java:131)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RecoveryPopUpNegativeTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:35:53 AMT" name="tearDown" finished-at="2022-07-17T15:35:53 AMT" duration-ms="121" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:35:53 AMT" name="setup" finished-at="2022-07-17T15:35:57 AMT" duration-ms="3699" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="goToLoginPopUp()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:35:57 AMT" name="goToLoginPopUp" finished-at="2022-07-17T15:35:57 AMT" duration-ms="446" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- goToLoginPopUp -->
        <test-method signature="RecoveryPopUpNegativeTest(java.lang.String)[pri:5, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:35:57 AMT" name="RecoveryPopUpNegativeTest" description="Validate on Password recovery pop up functionality of Recovery with invalid credentials" data-provider="invalidRecoveryData" finished-at="2022-07-17T15:36:39 AMT" duration-ms="41773" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[ g.babloyan@iqsoft.am]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RecoveryPopUpNegativeTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:36:39 AMT" name="tearDown" finished-at="2022-07-17T15:36:39 AMT" duration-ms="113" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:36:39 AMT" name="setup" finished-at="2022-07-17T15:36:42 AMT" duration-ms="2676" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="goToLoginPopUp()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:36:42 AMT" name="goToLoginPopUp" finished-at="2022-07-17T15:36:42 AMT" duration-ms="459" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- goToLoginPopUp -->
        <test-method signature="RecoveryPopUpNegativeTest(java.lang.String)[pri:5, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:36:42 AMT" name="RecoveryPopUpNegativeTest" description="Validate on Password recovery pop up functionality of Recovery with invalid credentials" data-provider="invalidRecoveryData" finished-at="2022-07-17T15:37:23 AMT" duration-ms="40835" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[g.babloyan@iqsoft.am ]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RecoveryPopUpNegativeTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:37:23 AMT" name="tearDown" finished-at="2022-07-17T15:37:23 AMT" duration-ms="127" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:37:23 AMT" name="setup" finished-at="2022-07-17T15:37:26 AMT" duration-ms="3048" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="goToLoginPopUp()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:37:26 AMT" name="goToLoginPopUp" finished-at="2022-07-17T15:37:27 AMT" duration-ms="413" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- goToLoginPopUp -->
        <test-method signature="RecoveryPopUpNegativeTest(java.lang.String)[pri:5, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:37:27 AMT" name="RecoveryPopUpNegativeTest" description="Validate on Password recovery pop up functionality of Recovery with invalid credentials" data-provider="invalidRecoveryData" finished-at="2022-07-17T15:38:08 AMT" duration-ms="41371" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[g .babloyan@iqsoft.am]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RecoveryPopUpNegativeTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:38:08 AMT" name="tearDown" finished-at="2022-07-17T15:38:08 AMT" duration-ms="127" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:38:08 AMT" name="setup" finished-at="2022-07-17T15:38:11 AMT" duration-ms="2372" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="goToLoginPopUp()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:38:11 AMT" name="goToLoginPopUp" finished-at="2022-07-17T15:38:11 AMT" duration-ms="552" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- goToLoginPopUp -->
        <test-method signature="RecoveryPopUpNegativeTest(java.lang.String)[pri:5, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:38:11 AMT" name="RecoveryPopUpNegativeTest" description="Validate on Password recovery pop up functionality of Recovery with invalid credentials" data-provider="invalidRecoveryData" finished-at="2022-07-17T15:38:52 AMT" duration-ms="40838" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[g. babloyan@iqsoft.am]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RecoveryPopUpNegativeTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:38:52 AMT" name="tearDown" finished-at="2022-07-17T15:38:52 AMT" duration-ms="122" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:38:52 AMT" name="setup" finished-at="2022-07-17T15:38:55 AMT" duration-ms="3190" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="goToLoginPopUp()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:38:55 AMT" name="goToLoginPopUp" finished-at="2022-07-17T15:38:56 AMT" duration-ms="387" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- goToLoginPopUp -->
        <test-method signature="RecoveryPopUpNegativeTest(java.lang.String)[pri:5, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:38:56 AMT" name="RecoveryPopUpNegativeTest" description="Validate on Password recovery pop up functionality of Recovery with invalid credentials" data-provider="invalidRecoveryData" finished-at="2022-07-17T15:39:37 AMT" duration-ms="41307" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[g.babloyan@iqsoft.a m]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RecoveryPopUpNegativeTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:39:37 AMT" name="tearDown" finished-at="2022-07-17T15:39:37 AMT" duration-ms="120" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:39:37 AMT" name="setup" finished-at="2022-07-17T15:39:40 AMT" duration-ms="2784" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="goToLoginPopUp()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:39:40 AMT" name="goToLoginPopUp" finished-at="2022-07-17T15:39:41 AMT" duration-ms="524" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- goToLoginPopUp -->
        <test-method signature="RecoveryPopUpNegativeTest(java.lang.String)[pri:5, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:39:41 AMT" name="RecoveryPopUpNegativeTest" description="Validate on Password recovery pop up functionality of Recovery with invalid credentials" data-provider="invalidRecoveryData" finished-at="2022-07-17T15:40:02 AMT" duration-ms="21614" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[g_babloyan@mail.com]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at testCases.PasswordRecoveryTest.RecoveryPopUpNegativeTest(PasswordRecoveryTest.java:131)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RecoveryPopUpNegativeTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:40:03 AMT" name="tearDown" finished-at="2022-07-17T15:40:03 AMT" duration-ms="112" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:40:03 AMT" name="setup" finished-at="2022-07-17T15:40:05 AMT" duration-ms="2474" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="goToLoginPopUp()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:40:05 AMT" name="goToLoginPopUp" finished-at="2022-07-17T15:40:06 AMT" duration-ms="417" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- goToLoginPopUp -->
        <test-method signature="RecoveryPopUpNegativeTest(java.lang.String)[pri:5, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:40:06 AMT" name="RecoveryPopUpNegativeTest" description="Validate on Password recovery pop up functionality of Recovery with invalid credentials" data-provider="invalidRecoveryData" finished-at="2022-07-17T15:40:47 AMT" duration-ms="40837" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[gor@babloyan@mail.ru]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RecoveryPopUpNegativeTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:40:47 AMT" name="tearDown" finished-at="2022-07-17T15:40:47 AMT" duration-ms="119" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="finish()[pri:0, instance:testCases.PasswordRecoveryTest@1033576a]" started-at="2022-07-17T15:40:47 AMT" name="finish" finished-at="2022-07-17T15:40:47 AMT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- finish -->
      </class> <!-- testCases.PasswordRecoveryTest -->
    </test> <!-- craftBetAutomation -->
  </suite> <!-- Default Suite -->
</testng-results>
